### PR TITLE
Add Test yamls

### PR DESCRIPTION
**Pull request by vyankyGH**

### CODE REVIEW

File: best-practices/require_pod_requests_limits/require_pod_requests_limits.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ metadata:
       and 'resources.limits.memory' per pod. If a namespace level request or limit is specified, 
       defaults will automatically be applied to each pod based on the 'LimitRange' configuration.
 spec:
-  validationFailureAction: audit
+  validationFailureAction: enforce
```
Comment (JimBugwadia) [inline best-practices/require_pod_requests_limits/require_pod_requests_limits.yaml:14]: any reason why this was changed to `enforce`? For most of the sample policies, we keep as `audit` and allow the user to set as enforce if they wish.

File: best-practices/require_probes/require_probes.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ metadata:
       are running or need to be restarted. A `readinessProbe` is used by services 
       and deployments to determine if the pod is ready to receive network traffic.
 spec:
-  validationFailureAction: audit
+  validationFailureAction: enforce
```
Comment (JimBugwadia) [inline best-practices/require_probes/require_probes.yaml:15]: any reason why this was changed to enforce? For most of the sample policies, we keep as audit and allow the user to set as enforce if they wish.

File: best-practices/restrict_image_registries/restrict_image_registries.yaml [hunk 3]
```diff
@@ -8,7 +8,8 @@ metadata:
       Images from unknown registries may not be scanned and secured. 
       Requiring use of known registries helps reduce threat exposure.
 spec:
-  validationFailureAction: audit
+  background: false
+  validationFailureAction: enforce
```
Comment (JimBugwadia) [inline best-practices/restrict_image_registries/restrict_image_registries.yaml:12]: any reason why this was changed to enforce? For most of the sample policies, we keep as audit and allow the user to set as enforce if they wish.

**Review by JimBugwadia — COMMENTED** [review]
see questions in the comments...

File: best-practices/require_pod_requests_limits/require_pod_requests_limits.yaml (hunk 1 above)
Comment (vyankyGH) [reply best-practices/require_pod_requests_limits/require_pod_requests_limits.yaml:14]: Not any specific reason. I was testing using enforce. I will revert back to audit.

File: best-practices/restrict_image_registries/restrict_image_registries.yaml (hunk 3 above)
Comment (vyankyGH) [reply best-practices/restrict_image_registries/restrict_image_registries.yaml:12]: Not any specific reason. I was testing using enforce. I will revert back to audit.

File: pod-security/default/disallow-host-namespaces/test.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  -  resource.yaml
+results:
+  - policy:  disallow-host-namespaces
+    rule:  host-namespaces
+    resource: host-namespaces-net
+    status: fail
+  - policy:  disallow-host-namespaces
+    rule:  host-namespaces
+    resource: host-namespaces-pid
+    status: fail
```
Comment (realshuting) [inline pod-security/default/disallow-host-namespaces/test.yaml:14]: Looks like the result for `host-namespaces-ipc` is missing.

File: pod-security/restricted/restrict-seccomp/test.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    resource:  seccomp-pod
+    status: fail
+  - policy:  restrict-seccomp
+    rule: seccomp
+    resource: seccomp-ctnr
+    status: pass
+  - policy:  restrict-seccomp
+    rule: seccomp
+    resource: seccomp-init-ctnr
+    status: pass
```
Comment (realshuting) [inline pod-security/restricted/restrict-seccomp/test.yaml:18]: Why `seccomp-ctnr` and `seccomp-init-ctnr` passed?

**Review by realshuting — CHANGES_REQUESTED** [review]
Thanks @vyankyGH , it looks good overall. Added a few comments.

**Review by realshuting — APPROVED** [review]

**Pull request merged**